data.py

```diff
@@ -82,26 +98,24 @@ def populate(oxigraph_url=None, write_to_db=True):
     # TODO - need to iterate all files in ./edtions not just the one
 
     # Load from disk
-    editions_source_path = Path(subbed_metadata_store_content_path / "editions/cpih_2022-01.json")
-    with open(editions_source_path) as f:
-        editions_source_dict = json.load(f)
+    editions_source_dict = subbed_metadata_store_content_path / "editions"
 
     # Validate then add to graph
     # TODO - add schema validation
-    g += Graph().parse(data=json.dumps(set_context(editions_source_dict)), format="json-ld")
+    schemas.Editions(**editions_source_dict)
```

Comment:
so this won't work, at the point you run it `editions_source_dict` is still just a `Path` object not a dictionary.

given the pattern you're using with `process_json_files` I would pass the schema in as an argument so you can validate after we json load and attach context in that function.